lib/mix/lib/mix/compilers/elixir.ex

```diff
@@ -724,7 +850,7 @@ defmodule Mix.Compilers.Elixir do
         purge_old_manifest(compile_path, data)
 
       _ ->
-        {[], [], %{}, nil}
+        {[], [], %{}, nil, nil, nil}
```

Comment:
Should we change this function to return a map?

Reply:
Yeah, a map or a record. I will add to my todo list.